charts/civil-ccd/Chart.yaml

```diff
@@ -30,39 +30,14 @@ dependencies:
     version: 2.3.0-alpha
     repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
     condition: idam-pr.enabled
-  - name: camunda-bpm
```

Comment:
This whole file doesn't match master. Here we have Camunda bpmn as false thats why your PR is failing. Please make it same as master. I dont believe as part of this ticket you need to make any changes in this yaml